peekingduck/pipeline/nodes/input/utils/preprocess.py

```diff
@@ -52,3 +52,19 @@ def mirror(frame: np.array) -> np.array:
     Mirrors a video frame.
     '''
     return cv2.flip(frame, 1)
+
```

Comment:
If `set_res` is no longer used, this function can be removed.

Reply:
removed.